### PR TITLE
Update cs-guide.md with courses

### PR DESCRIPTION
**Pull request by dsouvage**
Getting the ball rolling on adding courses to the guide, maybe we could have individual pages with reviews for each course down the line also. Might be crazy though.

### CODE REVIEW

File: src/assets/cs-guide.md [hunk 1]
```diff
@@ -29,8 +29,84 @@ Please add yourself to this list (link is optional) if you've contributed to thi
...
+| APCO-1P01 | Fluency with Technology | ✅ | ✅ | ✅ | 💛| | 
+| APCO-1P30 | Programming for Interactive Media | ❌ | ✅ | ❌ | 💛| |
+| COSC-1P50 | Integrity and Literacy in the Information Age | ✅ | ✅ | ✅ | 💛❤️| |
+| APCO-1P93 | Applied Programming |  ❌ | ✅ |  ❌ | 💛| APCO-1P00 or 4U Mathematics credit |
+| APCO-2P11 | Personal Computers and Networks |  ❌ | ✅ |  ❌ | 💛| APCO-1P01 |
+| COSC-2P89 | Internet Technologies |  ❌ | ✅ |  ❌ | 💛💜| APCO 1P00 (minimum 60 percent) and 1P01 (minimum 60 percent) or COSC 1P03 (minimum 60 percent) |
+| COSC-3P94 | Human Computer Interaction |  ❌ | ✅ |  ❌ | 💛💜| Two Credits from either IASC/APCO/COSC |
+
+
+## School Resources
```
Comment (kindeep) [inline src/assets/cs-guide.md:88]: It looks like School Resources and goodies were repeated?

Comment (kindeep) [reply src/assets/cs-guide.md:88]: Actually looks like all previous content was somehow repeated a second time

Comment (firefelix) [reply src/assets/cs-guide.md:88]: whoops, might have to do with the way I edited it, can you resolve this from here or should I submit a new PR

Comment (firefelix) [reply src/assets/cs-guide.md:55]: i think i fixed it

**Review by kindeep — APPROVED** [review]
Looks good

**Pull request merged**